tests/page_objects/project_search_page.py

```
@@ -0,0 +1,60 @@
+            return True
+        except:
+            return False
+
+    def get_no_results_message(self):
+        """Get the no results message."""
+        try:
+            message_element = self.wait.until(
+                EC.presence_of_element_located(ProjectSearchLocators.NO_RESULTS_MESSAGE)
+            )
+            return message_element.text
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)